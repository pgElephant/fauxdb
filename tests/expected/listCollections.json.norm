{
  "cursor": {
    "firstBatch": [
      {
        "name": "test",
        "type": "collection"
      }
    ],
    "id": 0,
    "ns": "admin.$cmd.listCollections"
  },
  "ok": 1
}
