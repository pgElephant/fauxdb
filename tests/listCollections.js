db.runCommand({listCollections: 1})
