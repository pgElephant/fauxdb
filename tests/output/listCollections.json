{
  ok: 1,
  cursor: {
    id: Long('0'),
    ns: 'admin.$cmd.listCollections',
    firstBatch: [ { name: 'test', type: 'collection' } ]
  }
}
