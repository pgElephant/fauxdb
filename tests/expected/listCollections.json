{
  "ok": 1,
  "cursor": {
    "id": 0,
    "ns": "admin.$cmd.listCollections",
    "firstBatch": [
      {
        "name": "test",
        "type": "collection"
      }
    ]
  }
}
